documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -237,7 +237,7 @@ export default config;
 /// file: wrangler.toml
 ---site.bucket = ".cloudflare/public"---
 +++assets.directory = ".cloudflare/public"
-assets.binding = "ASSETS"+++
+assets.binding = "ASSETS" # Add this binding only if you have Worker scripts+++
```

Comment:
I wonder if this works so that the comment doesn't get highlighted
```suggestion
assets.binding = "ASSETS"+++ # Add this binding only if you have Worker scripts
```

Reply:
I tried this but it highlighted the full line anyways, so I reverted it the original version with the `+++` at the end.

Reply:
Ah no worries. There's a PR to try and fix the highlighting https://github.com/sveltejs/svelte.dev/pull/1480 but it's not ready yet and this isn't that important